models/events/webhook/github/GithubPush.js

```diff
@@ -0,0 +1,5 @@
+module.exports = require('../../../Model').extend({
+  properties: {
+    compare: String
```

Comment:
What is `compare` supposed to be? Is it a place holder or is that supposed to be actual data from a push?